modules/dnn/src/model.cpp

```diff
@@ -209,15 +215,33 @@ ClassificationModel::ClassificationModel(const Net& network)
     // nothing
 }
 
+ClassificationModel& ClassificationModel::setSoftMax(bool apply)
+{
+    applySoftmax = apply;
+    return *this;
+}
+
+bool ClassificationModel::getSoftMax() const
+{
+    return applySoftmax;
+}
+
 std::pair<int, float> ClassificationModel::classify(InputArray frame)
 {
-    std::vector<Mat> outs;
+    std::vector<cv::Mat> outs;
```

Comment:
No need to use `cv::` in OpenCV sources - they are usually are part of cv namespace already.

Reply:
I removed the explicit specification of namespace `cv::`.